api/grpc_server.go

```diff
@@ -246,14 +247,14 @@ func (s SpacemeshGrpcService) GetAccountTxs(ctx context.Context, txsSinceLayer *
 	if err != nil {
 		return &pb.AccountTxs{}, err
 	}
-	if txsSinceLayer.StartLayer > txsSinceLayer.EndLayer {
+	if txsSinceLayer.StartLayer > uint64(s.Tx.LatestLayer()) {
```

Comment:
maybe call `LatestLayer` once and store the value since you use it again ?

Reply:
tricky quesiton... I usually don't try to optimize what the compiler should